### DISCUSSION TITLE
posts/2023/07/zhejiang-study/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#13] [General]
# posts/2023/07/zhejiang-study/

https://guozheng.netlify.app/posts/2023/07/zhejiang-study/

**Comment by liechi** [comment]
我见过的最惊心动魄的人类学研究来自毛泽东在建国前写的一系列文章，比如中国社会各阶层分析，湖南农民运动考察等。虽然文章里略去了细节，但其对当时社会各阶层民众状态的精确把握体现了作者在实地考察中做了极为细致的观察和分析。而且，作者不止步于观察描述分析，还将这些观察和分析总结起来指导革命实践，并取得了胜利。要问人类学有什么用，我觉得毛泽东提供了一个雄辩的回答，虽然很少有人觉得他做的是人类学研究。

**Reply by residualsun1** [reply]
你这么一说，我也觉得毛泽东做过的研究是很珍贵的田野调查。大一上《社会学研究方法》时，里面就提过毛泽东的几项研究。以前没什么感受，一方面是自己完全没有田野经验，另一方面也是停留在对概念、理论的一知半解和自说自话的狭隘认知里。可人类学是极其讲究经验的学科，如梁永佳老师说的：

> 社会科学往往都告诉你应该怎么样（有预设好的，或期望、或模型），但人类学一反惯常话语，告诉你生活原本的模样，生活就是生活本身。

然后就是本土化的重要性。梁老师今天在讲座上就讲到自己的研究追求，要做一件疯狂的事情：用中国先秦的思想来研究人类学。中国的人类学鲜有突破，也缺少大家，似乎是大家总在西方的话语里走，但是抛开我们上下五千年的底蕴，我们也无法进入到古希腊与古罗马，他还打趣地和我们说，**他先疯狂一把，我们暂时不要疯狂**。我想，将目光放得近一点，如毛泽东等并非人类学家的前人，他们对近代社会的研究都根植社会历史现实，里面应当藏有属于我们自己的智慧。

**Reply by liechi** [reply]
用先秦思想研究人类学有什么优势吗？依据是什么？不知道梁老师有没有说明？

我对社会科学不熟悉，但是在零零星星的接触中也感到一种趋势，就是越来越多的学者试图从古代思想中获得创新的灵感。一个比较有影响的例子是赵汀阳借鉴我国古代的”天下体系“来构建新的国际关系。一方面，我觉得从他人，包括古人，的智慧中学习是必要的---但凡有可借鉴处就不必重新发明轮子；另一方面，我又对这样的做法存有疑虑---这样的借鉴到底是否符合实际？还是一种思想懒惰的取巧，甚至是挂羊头卖狗肉的伪创新？如果古代智慧能跟现在的实践融合萌发新的生机，那我是非常乐意看到的。只是也觉得这怕是不容易的。

以前我觉得理解什么是人类学有些困难，但现在我倒是觉得理解什么是”非人类学“有些困难。按我比较浅薄的理解，只要是跟人的活动相关的，万物皆可人类学。比如解放军手册记录了在战争期间总结出来的一系列后勤保障原则，如何最大限度地利用有限物资，如何吃饭，睡觉，在哪里挖卫生壕等等，我觉得就是很好的特殊环境下的人类学研究。高更那些记录了塔希提人的生活状态的作品在我看来也是优秀的人类学文献，特别是那幅[关于人间三问的巨作](https://zh.wikipedia.org/zh-cn/%E4%BF%9D%E7%BE%85%C2%B7%E9%AB%98%E6%9B%B4#/media/File:Paul_Gauguin_142.jpg)，包含了塔希提人的生老病死，衣食住行，宗教信仰，简直就是雄文一篇。我对人类学的这种看法是一种比较极端的简化，容易贻笑大方，只是想到这里，就不揣冒昧地写下来了。

**Reply by residualsun1** [reply]
这两天忙成陀螺了π_π…梁老师没有具体说，不过今天下午有一个“以《老子》解读美拉尼西亚社会”的讲座，听完后我回来反馈一下( ˙˘˙ )。

**Reply by residualsun1** [reply]
> 用先秦思想研究人类学有什么优势吗？依据是什么？不知道梁老师有没有说明？

@liechi 梁老师有大概提及到的，不过我不大记得了。留在记忆中的大概是，如果真的要在人类学中做出比较大的突破，那应当很熟悉西方整一个历史（追溯到希腊和罗马），这些在西方学校的训练中可能就比较多。我们现在的社会科学和人文学科的学习的主流方向还是以西方为主，但由于我们语言和文化的差异，在梁老师看来，我们不可能在西方的人类学中做出真正突破，出现一两位人类学大家。但相对的，我们也有我们上下五千年的历史文化，他想试试用我们的文化底蕴去进行解读会发现什么（所以这是“疯狂”的）。另外，用老子解读人类学也受到一些偶然因素的影响，似乎是有西方人类学家用中国混沌和阴阳的概念来解释库拉圈，这启发梁老师将老子的思想和库拉圈的研究联系起来。

列驰，如果你感兴趣的话推荐到《社会学研究》上看看梁老师写的这篇[“以《老子》解读美拉尼西亚社会”](http://shxyj.ajcass.org/Admin/UploadFile/Issue/blipyfnb.pdf)。主要是用《老子》中“贵货不积”的思想来理解库拉圈交换的情况。梁老师花了一个多小时和我们进行了介绍，不过我之前不了解库拉圈，所以听下来有点懵。如果要补充库拉圈的知识，可以阅读马林诺夫斯基写的《西太平洋上的航海者》。

另，“以《老子》解读美拉尼西亚社会”有英文版，名字是“”Esteeming Goods for non-Accumulation, Small Realm with Few People: interpreting Kula with Laozi"，内容很不一样，刊登在美国著名的人类学期刊“American Anthropologist”。听梁老师说，上一次在这个期刊上发表文章的中国人类学者还是在六十多年前。不过在文章评议时，他还是被专门研究库拉圈的国外学者严厉地批评了哈哈。

**Reply by residualsun1** [reply]
> 按我比较浅薄的理解，只要是跟人的活动相关的，万物皆可人类学。

@liechi 从某种程度上讲，我觉得确实可以理解为“万物皆可人类学”。有饮食人类学啦、影视人类学啦、生态人类学啦等等。在这里和你分享一下我们和梁老师圆桌讨论时得出的看法：

梁老师觉得人类学是一门“经验哲学”。哲学可以提出好问题，但无法解决问题，但是经验的人类学可以基于哲学的问题提供可能的解决路径。人类学的独特方式在于，我们可以**用（土著的）异文化去理解我们现有的文化，理解我们自己还有我们的世界**。当时他举了一个例子，让我头一次感到胆战心惊。他从莫斯《礼物》这本书出发，认为“社会”本质上是“**我期待**”，不是卢梭说的契约论，也不是霍布斯说的利维坦，具体上他娓娓道来了很多例证，但我没能一一记清，不过我不会忘记到结论梁老师说到“**社会科学原来那一套的说法都是错的**”时，我整个人都懵了，有一种心中一惊和起鸡皮疙瘩的感觉，我想不到一点反驳的理由。

最后我给自己的答案是，人类学的独特在于“**像土著一样思考**”，从异文化的角度出发，对地方性的智慧进行**文化翻译**。这对应了过去兵哥和我说的“讲故事”（因此我过去觉得人类学就是一门讲故事的学科）。所以我觉得你的理解没有错，人类学就是一门与人所有相关活动有关的学科，但它独特在于，能看见不同文化的地方性智慧，并尝试以“我们”能听懂的方式叙述出来。

另，人类学非常重视**经验**和**生活本身**，所以你举到的例子都是很重要的材料。然后不知道你有没有听说过**非虚构写作**和**民族志**这两个概念？梁老师就觉得，契诃夫的小说可以说是非常优秀的民族志，换句话说，优秀的小说家就像人类学家（当然前提应该写的是非虚构写作）。他就给我们极力推荐契诃夫的《苦恼》，还有董桥的一篇短篇小说《云姑》。我想，如果还要体现出“人类学”的独特，以区别开文学小说创作，那应该就是要有**实际的田野调查**和相关**人类学的理论**吧。（两个进行文化翻译的重要工具）

---

附上一个我很喜欢的袁长庚老师对人类学的理解，他觉得人类学是一门关心**人的脆弱性**的学科。哈哈，我单纯就很喜欢这个表达，应该还有很丰富的内容可以阐释。他的研究也很有意思，关注的是年轻人的情感，还有生死观念这些。

**Reply by liechi** [reply]
你提到的非虚构写作和民族志这两个概念我都不熟悉。不过我很赞同小说，优秀的小说，是极好的民族志这个看法。上次我提到了毛泽东，这次我想到的是鲁迅。鲁迅曾在一篇分享写小说的经验的文章里提到，写小说的人需要有一双会观察的眼睛（这点跟人类学家相似）。我的理解是，优秀的小说家会描写一些我们已经习以为常，但被人用笔写出来后自己读来也绝震撼的生活细节---比如鲁迅反复提到的“看客”。生活中到处都是看客，但很少有人觉察到自己作为看客的状态。如此种种。鲁迅描述的看客，在后来轰轰烈烈的革命中却不再是看客，成为了中流砥柱。这是为什么？在我看来，鲁迅和毛泽东是两个接力的“人类学家”，一个看病，一个治病。

**Reply by liechi** [reply]
> 列驰，如果你感兴趣的话推荐到《社会学研究》上看看梁老师写的这篇“以《老子》解读美拉尼西亚社会”。

嗯，我会的。

我对人类学的了解极为粗浅，觉得继续深入的话有点吃力了。等以后有了新的感悟了或许再来继续。迄今为止，受益匪浅。

对了，我的名中的那个字是“弛”：）

**Reply by residualsun1** [reply]
> 迄今为止，受益匪浅。

我也是门外汉，不过跟你分享我感受的过程也是我的获益匪浅。

> 对了，我的名中的那个字是“弛”：）

我记住了！

**Comment by DylanYu233** [comment]
好好好

**Comment by poemboy1** [comment]
同第三届人类学训练营路过哈哈哈

**Reply by residualsun1** [reply]
你好你好！相遇就是缘分，欢迎来小破站இ௰இ，（(っ °Д °;)っ可是你是怎么发现这里的……）

**Reply by poemboy1** [reply]
一点都不破，很丰富。GIt上搜索文科找工作，看到评论，你好像也回复了，就看到博客哈哈哈

**Reply by residualsun1** [reply]
感谢你的肯定(‾◡◝)!! 我感觉一般用 GitHub 的多是理工科出身的用户，他们一般好像不对社会学、人类学很有兴趣，没想到我们会以这样的方式对话。

**Reply by poemboy1** [reply]
哈哈哈，好奇你这个博客怎么做的，看了好多教程都做不来

**Reply by residualsun1** [reply]
我之前用 Rstudio 学习数据分析，无意中发现可以用 R 包 [Blogdown](https://github.com/rstudio/blogdown) 搭建网站，参考论坛统计之都的文章《[用 R 语言的 blogdown+hugo+netlify+github 建博客](https://cosx.org/2018/01/build-blog-with-blogdown-hugo-netlify-github/)》来做的。但是这可能对没使用过 Rstudio 的用户来说比较麻烦，一般来说可以直接在本地部署，Hugo 本身就是这样的，是一位 R [码农](https://github.com/yihui)将 Hugo 的功能接入 R 语言。

可以看看[郝鸿涛](https://github.com/hongtaoh)的这篇博文《[如何零基础免费搭建个人网站](https://hongtaoh.com/cn/2021/03/02/personal-website-tutorial/)》，或者《[如何零基础免费搭建个人网站 (2024 更新版)](https://hongtaoh.com/cn/2024/03/22/personal-website-tutorial/)》，这些不涉及 Rstudio，而且评论区也有很多疑惑和解答。同时，也可以结合 AI。可以在 Hugo的网站里先挑选自己喜欢的主题，然后再回头照着帖子来设置。

由于我之前是用 Rstudio 来建站，也没有及时记录自己的建站过程，所以一下子还没办法说得很详细。你可以了解《[用 R 语言的 blogdown+hugo+netlify+github 建博客](https://cosx.org/2018/01/build-blog-with-blogdown-hugo-netlify-github/)》和《[如何零基础免费搭建个人网站](https://hongtaoh.com/cn/2021/03/02/personal-website-tutorial/)》这两篇文章，看看哪一个更适合你。

**Reply by residualsun1** [reply]
如果决定用 Rstudio 和 Blogdown 包来搭建，过程中遇到问题可以来直接问我^_^ ！

**Reply by poemboy1** [reply]
好的！感谢！！我慢慢琢磨

**Reply by residualsun1** [reply]
不客气！

对了，如果你更想专注于写作的话，也可以找一些设计精美、功能齐全的博客托管平台，就是每个月要付一些钱。

我知道一个 [Typlog](https://typlog.com)，一个比较成熟的网站样例可以参考博主[太隐](https://wangyurui.com)。

「少数派」上也有对 Typlog 的具体介绍，可参考：《[功能全、价格低、操作简单，它是个人建站和播客创作的新选择](https://sspai.com/post/56113)》。

自己搭建网站的话要学习一些静态网页的技术，了解一定的 html、css 和 java（不过前期也可以先不管，以原主题设置为主，后期一边写一边学着调整，我就是这样的，一边写一边改，有时候问网友，现在问 AI 为主），比较麻烦，但是掌控感可能强一些，学习的过程成就感也很强，基本全免费（除了考虑特殊的域名）。而找一些他人设计的网站托管也是一个可行的选择，功能啥的都设计好了，比较容易操作，就是可能有点点小贵。

**Reply by poemboy1** [reply]
好的，感谢，感谢推荐，太隐博客确实不错，但我还没有想那么多修饰的。
昨晚用hugo弄了半天，总是出错误。我现在试试rstudio哈哈哈
